app/dashboard/utils.py

```diff
@@ -0,0 +1,194 @@
+    standardbounties_addr = getStandardBountiesContractAddresss(network)
+    bounty_abi = json.loads(standardbounties_abi)
+    getBountyContract = web3.eth.contract(standardbounties_addr, abi=bounty_abi)
+    return getBountyContract
+
+
+def get_bounty(bounty_enum, network):
+    try:
+        ipfs = getIPFS()
```

Comment:
Same comment with this `try` statement.  I try to make the `try` statement pretty thin to clearly show what is being tested.

Reply:
updated